examples/src/main/java/com/mapbox/navigation/examples/ui/CustomUIComponentStyleActivity.kt

```diff
@@ -0,0 +1,560 @@
+        super.onDestroy()
+        mapView.onDestroy()
+
+        mapboxNavigation.unregisterLocationObserver(locationObserver)
+        mapboxNavigation.unregisterTripSessionStateObserver(tripSessionStateObserver)
+        mapboxNavigation.unregisterRouteProgressObserver(routeProgressObserver)
+        mapboxNavigation.unregisterBannerInstructionsObserver(bannerInstructionObserver)
+        mapboxNavigation.unregisterVoiceInstructionsObserver(voiceInstructionsObserver)
+
+        mapboxNavigation.stopTripSession()
+        mapboxNavigation.onDestroy()
+
+        speechPlayer.onDestroy()
+    }
+
+    override fun onSaveInstanceState(outState: Bundle) {
+        super.onSaveInstanceState(outState)
+        mapView.onSaveInstanceState(outState)
+    }
+
+    override fun onMapReady(mapboxMap: MapboxMap) {
+        Timber.d("onMapReady")
+        this.mapboxMap = mapboxMap
+        mapboxMap.moveCamera(CameraUpdateFactory.zoomTo(15.0))
+
+        mapboxMap.addOnMapLongClickListener { latLng ->
+            Timber.d("onMapLongClickListener position=%s", latLng)
+            destination = latLng
+            locationComponent?.lastKnownLocation?.let { originLocation ->
+                mapboxNavigation.requestRoutes(
+                    RouteOptions.builder().applyDefaultParams()
+                        .accessToken(Utils.getMapboxAccessToken(applicationContext))
+                        .coordinates(originLocation.toPoint(), null, latLng.toPoint())
+                        .alternatives(true)
+                        .profile(DirectionsCriteria.PROFILE_DRIVING_TRAFFIC)
+                        .build(),
+                    routesReqCallback
+                )
+            }
+            true
+        }
+
+        mapboxMap.setStyle(Style.MAPBOX_STREETS) { style ->
+            locationComponent = mapboxMap.locationComponent.apply {
+                activateLocationComponent(
+                    LocationComponentActivationOptions.builder(
+                        this@CustomUIComponentStyleActivity, style
+                    ).build()
+                )
+                cameraMode = CameraMode.TRACKING
+                isLocationComponentEnabled = true
+            }
+
+            navigationMapboxMap = NavigationMapboxMap(mapView, mapboxMap).apply {
+                addOnCameraTrackingChangedListener(cameraTrackingChangedListener)
+                addProgressChangeListener(mapboxNavigation)
+                setCamera(DynamicCamera(mapboxMap))
+            }
+        }
+    }
+
+    // InstructionView Feedback Bottom Sheet listener
+    override fun onFeedbackSelected(feedbackItem: FeedbackItem?) {
+        feedbackItem?.let { feedback ->
+            mapboxMap?.snapshot { snapshot ->
+                alertView.showFeedbackSubmitted()
+                MapboxNavigation.postUserFeedback(
+                    TelemetryUtils.obtainUniversalUniqueIdentifier(),
+                    feedback.feedbackType, feedback.description, encodeSnapshot(snapshot)
```

Comment:
If you look at the signature of `MapboxNavigation.postUserFeedback()` it looks like as follows:
```
@JvmStatic
        fun postUserFeedback(
            @FeedbackEvent.FeedbackType feedbackType: String,
            description: String,
            @FeedbackEvent.FeedbackSource feedbackSource: String,
            screenshot: String?
        )
```
Looking at the signature, looks like the values that are being passed are not correct.

Reply:
🤔 interesting! The code initially came from driver app code when @Guardiola31337 and @langsmith and I integrated the feedback UI into the driver app. Maybe something changed after that.

But anyway, good catch @abhishek1508 !

I will make a change. Also will cut a ticket at driver app since it uses the same code.

Reply:
This also needs to be fixed in the UI SDK as well. I first observed this issue there. 
cc @JunDai

Reply:
Opening a separate ticket to fix it in the UI SDK cc @JunDai

Reply:
https://github.com/mapbox/mapbox-navigation-android/issues/2770

Reply:
> This also needs to be fixed in the UI SDK as well. I first observed this issue there.
> cc @JunDai

The UI SDK doesn't have this issue. The code in InstructionViewActivity is an example and I have updated it in this PR as well.